src/Apps/Order/Routes/Respond/index.tsx

```diff
@@ -62,7 +62,9 @@ export class RespondRoute extends Component<RespondProps, RespondState> {
 
   onContinueButtonPressed: () => void = () => {
     this.setState({ isCommittingMutation: true }, () => {
-      window.alert("You did a click!")
+      if (this.state.responseOption === "ACCEPT") {
+        this.props.router.push(`/orders/${this.props.order.id}/review/accept`)
+      }
```

Comment:
:+1:

Tests for this page were added in #1639 so you'll need to pull those changes in and then, for the 'ACCEPT' option only, replace the window.alert stuff with the above route change, and then a test assertion like this https://github.com/artsy/reaction/blob/c93f27bfbdf92ab0e8851e0ca4d4e89fe6745716/src/Apps/Order/Routes/__tests__/Review.test.tsx#L72

Reply:
Thanks for the tip on the test. I think I got the merge of index.tsx correct, but I'll definitely have to go add the test.

Reply:
We updated the corresponding test.